utils/generate_html_pages.sh

```diff
@@ -74,33 +77,13 @@ echo "</body>" >> index.html
 echo "</html>" >> index.html
 popd
 
-# Generate SRG Mapping Tables
-mkdir -p $PAGES_DIR/srg_mapping
-pushd $PAGES_DIR/srg_mapping
-touch index.html
-echo "<html>" > index.html
-echo "<header>" >> index.html
-echo "<h1>SRG Mapping Tables</h1>" >> index.html
-echo "</header>" >> index.html
-echo "<body>" >> index.html
-echo "<ul>" >> index.html
-srg_products="rhel9" # space separated list of products
-for product in $srg_products
-do
-echo "<li><a href=\"srg-mapping-${product}.html\">srg-mapping-${product}.html</a></li>" >> index.html
-echo "<li><a href=\"srg-mapping-${product}.xlsx\">srg-mapping-${product}.xlsx</a></li>" >> index.html
-done
-echo "</ul>" >> index.html
-echo "</body>" >> index.html
-echo "</html>" >> index.html
-popd
-
 pushd $PAGES_DIR
 touch index.html
 echo "<html>" > index.html
```

Comment:
```suggestion
echo "<!DOCTYPE html>" > index.html
echo '<html lang="en">' >> index.html
echo '<meta charset="utf-8" />' >>
```
If we want this to be a valid HTML5 document.